src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1974,7 +1857,6 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
     gpr_free(glb_policy);
     return NULL;
   }
-  grpc_subchannel_index_ref();
```

Comment:
This line should not be removed -- it's unrelated to the PR being reverted here.

Reply:
Good catch.   Looks like this was an artifact of the upstream merge commit.